serverless/containers/how-to/secure-a-container.mdx

```diff
@@ -0,0 +1,115 @@
+---
+meta:
+  title: How to secure a container
+  description: This page explains how to secure a Serverless container
+content:
+  h1: How to secure a container
+  paragraph: This page explains how to  how to secure a Serverless container
+tags: containers secure private
+dates:
+  validation: 2023-02-01
+  posted: 2023-02-01
+categories:
+  - serverless
+---
+
+This page explains how to secure your container.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created a containers namespace](/serverless/containers/how-to/create-a-containers-namespace/)
+  - You have [deployed a container](/serverless/containers/how-to/deploy-a-container/)
+</Message>
+
+## Use Secrets to store access keys and sensitive information
+
+Instead of using environment variables (which are stored in clear text) for your containers, use Secrets. These are pieces of information that can be used via environment variables, but are encrypted in storage. 
+
+### Configure Secrets from the Scaleway Console
+
+1. Click **Containers** in the **Serverless** section of the side menu. The containers page displays.
+2. Click the relevant container namespace. 
+3. Click the name of the container for which you want to define Secrets.
+4. Click the **Deployment** tab.
+5. Scroll to the **Secrets** section of the page and click **Add a new secret**. Enter the key and value for your Secret. Repeat for additonal secrets.
+    <Message type="important">
+        Be careful when you type your Secrets. You will not be able to read the value in the console once submitted.
+    </Message>
+6. Click **Deploy a Container** to submit your Secrets and redeploy your container. 
+
+### Configure Secrets using the Serverless framework
+
+Add `secret` to your containers description (more information in the [plugin documentation](https://github.com/scaleway/serverless-scaleway-containers#security-and-secret-management)).
```

Comment:
link is dead, replace with : https://github.com/scaleway/serverless-scaleway-functions#security-and-secret-management

Reply:
```suggestion
Add `secret` to your containers description (more information in the [plugin documentation](https://github.com/scaleway/serverless-scaleway-functions#security-and-secret-management)).
```